ykrt/src/mt.rs

```diff
@@ -25,55 +22,29 @@ const PHASE_NUM_BITS: usize = 2;
 // whose value is a pointer to memory. By also making that tag 0b00, we allow that index to be
 // accessed without any further operations after the initial tag check.
 const PHASE_TAG: usize = 0b11; // All of the other PHASE_ tags must fit in this.
-const PHASE_COMPILED: usize = 0b00;
-const PHASE_TRACING: usize = 0b01;
-const PHASE_COUNTING: usize = 0b10; // The value specifies the current hot count.
-
-/// A `Location` is a handle on a unique identifier for control point position in the end-user's
-/// program (and is used by the `MT` to store data about that location). In other words, every
-/// position that can be a control point also needs to have one `Location` value associated with
-/// it. Note however that instances of `Location` may be freely cloned and shared across threads.
+const PHASE_COMPILED: usize = 0b00; // Value is a pointer to a malloc'd block containing a
```

Comment:
The traces are `mmap`'d, but it's probably be best to not specify how it was allocated, since that decision isn't ours (it's dynasm's).

Reply:
Agreed, I think the current vagueness is about right.